### PR TITLE
Videolisting feature

### PR DESCRIPTION
**Pull request by Pratik1005**
Added video filters which filter based on category

### CODE REVIEW

File: src/pages/Home/VideoListing.jsx [hunk 1]
```diff
@@ -1,33 +1,76 @@
...
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/videos");
+        setVideos(response.data.videos);
+        setLoader(false);
+        setCategoryVideos(response.data.videos);
+      } catch (err) {
+        console.error("video listing", err);
```
Comment (Ash-D23) [inline src/pages/Home/VideoListing.jsx:27]: have a setloader false in error handler also

File: src/App.css [hunk 2]
```diff
@@ -24,34 +24,56 @@ aside {
...
+  cursor: pointer;
+} */
+
+/* aside nav ul li:hover {
+  background-color: var(--menu-hover);
+} */
+
+/* aside nav ul li span.material-icons {
+  padding-right: 10px;
+} */
```
Comment (simran61) [inline src/App.css]: try removing extra comments

File: src/utils/convertViews.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const convertViews = (views) => {
+  switch (views.length) {
+    case 4:
```
Comment (Ash-D23) [inline src/utils/convertViews.js:3]: why so many empty cases, instead if else can be used if only two conditions

**Review by Ash-D23 — APPROVED** [review]
Looks clean

**Review by simran61 — COMMENTED** [review]
Try removing extra comments else it looks good.
Nice folder structure.

File: src/utils/convertViews.js (hunk 3 above)
Comment (Pratik1005) [reply src/utils/convertViews.js:3]: Its not empty cases, the code is same for that cases

**Pull request merged**